unicodetools/data/ucd/dev/DerivedCoreProperties.txt

```diff
@@ -3553,7 +3553,8 @@ E0100..E01EF  ; Case_Ignorable # Mn [240] VARIATION SELECTOR-17..VARIATION SELEC
 
 # Derived Property:   Changes_When_Lowercased (CWL)
 #  Characters whose normalized forms are not stable under a toLowercase mapping.
-#  For more information, see D139 in Section 3.13, "Default Case Algorithms".
+#  For more information, see the definition of "isLowercase(X)"
```

Comment:
I think definition numbers such as D139 are supposed to remain stable. @Ken-Whistler or @lianghai should know.